test/sql/detailed_profiler/test_detailed_profiler.test

```diff
@@ -0,0 +1,37 @@
+# name: test/sql/detailed_profiler/test_detailed_profiler.test
+# description: Detailed Profiler
+# group: [detailed_profiler]
+
+
+statement ok
+PRAGMA enable_profiling
+
+statement ok
+PRAGMA profiling_output='test.json'
+
+statement ok
+PRAGMA profiling_mode = detailed
+
+statement ok
+CREATE TABLE integers(i INTEGER)
+
+statement ok
+INSERT INTO integers VALUES (3)
+
+statement ok
+SELECT min (i + i) FROM integers
+
+statement ok
+CREATE TABLE exprtest (a INTEGER, b INTEGER)
+
+statement ok
+INSERT INTO exprtest VALUES (42, 10), (43, 100), (NULL, 1), (45, -1)
+
+statement ok
+SELECT min (a + a ) FROM exprtest
+
+statement ok
+SELECT a FROM exprtest WHERE a BETWEEN 43 AND 44
+
+statement ok
+SELECT CASE a WHEN 42 THEN 100 WHEN 43 THEN 200 ELSE 300 END FROM exprtest
```

Comment:
I would also like to see tests including parallelism (`test/sql/tpch/tpch_parallel_sf01.test_slow`).